.vscode/tasks.json

```diff
@@ -0,0 +1,29 @@
+{
+	"version": "2.0.0",
+	"tasks": [{
+		"label": "Start dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml up -d",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml up -d"
+		},
+		"group": "none",
+		"presentation": {
+		  "reveal": "always",
+		  "panel": "new"
+		}
+	  }, {
+		"label": "Stop dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml down",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml down"
+		},
+		"group": "none",
+		"presentation": {
+		  "reveal": "always",
+		  "panel": "new"
+		}
+	  }
```

Comment:
If you apply my suggestion regarding not running the docker stack in background (i.e. without `-d`), we could remove this task (or just leave it for completeness sake). Let me know :+1:

Reply:
Sounds good!